administrator/components/com_guidedtour/guidedtour.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<extension type="component" method="upgrade">
+<!-- 'version' attribute for extension tag is no longer used -->
+
+    <name>Guided Tour</name>
```

Comment:
Check the structure here: https://github.com/joomla-projects/soc21_guided-tour/blob/dev/administrator/components/com_banners/banners.xml#L3 and stick to them, also who is author (Joomla! Project) and the tags used.